unity/JetBrains.Rider.Unity.Editor/EditorPlugin/PluginEntryPoint.cs

```diff
@@ -133,18 +133,19 @@ private static void Init()
 
         riderProtocolController.Wire.Connected.WhenTrue(lifetime, connectionLifetime =>
         {
-          var protocol = new Protocol("UnityEditorPlugin", serializers, identities, MainThreadDispatcher.Instance,
-            riderProtocolController.Wire);
+          var protocol = new Protocol("UnityEditorPlugin", serializers, identities, MainThreadDispatcher.Instance, riderProtocolController.Wire);
           ourLogger.Log(LoggingLevel.VERBOSE, "Create UnityModel and advise for new sessions...");
           var model = new EditorPluginModel(connectionLifetime, protocol);
           AdviseUnityActions(model, connectionLifetime);
           AdviseModel(model);
           OnModelInitialization(new UnityModelAndLifetime(model, connectionLifetime));
           AdviseRefresh(model);
           
+          model.FullPluginPath.Advise(connectionLifetime, AdditionalPluginsInstaller.UpdateSelf);
```

Comment:
If there isn't a race because we don't set this property until UnityModel is set, maybe we need a comment on both sides, this seems like an easy place to actually create a race. Maybe we should provide a separate property IsEditorPluginReady for that?